runelite-client/src/main/java/net/runelite/client/plugins/emojis/EmojiPlugin.java

```diff
@@ -201,29 +201,4 @@ String updateMessage(final String message)
 
 		return Strings.join(messageWords, " ");
 	}
-
-	/**
-	 * Remove tags, except for &lt;lt&gt; and &lt;gt&gt;
-	 *
-	 * @return
-	 */
-	private static String removeTags(String str)
```

Comment:
Moving `removeFormattingTags` is fine, but should be in a separate commit since it is unrelated.

Reply:
Sure, removed.